src/Service/CaseEventHelper.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\CaseEntity;
+use App\Entity\CaseEvent;
+use App\Entity\CaseEventPartyRelation;
+use App\Entity\DigitalPost;
+use App\Entity\Party;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Security\Core\Security;
+
+class CaseEventHelper
+{
+    public function __construct(private EntityManagerInterface $entityManager, private Security $security)
+    {
+    }
+
+    public function createDigitalPostCaseEvent(CaseEntity $case, DigitalPost $digitalPost, array $recipients)
+    {
+        $caseEvent = new CaseEvent();
+
+        $caseEvent
+            ->setCaseEntity($case)
+            ->setCategory(CaseEvent::CATEGORY_OUTGOING)
+            ->setSubject(CaseEvent::SUBJECT_HEARING_CONTRADICTIONS_BRIEFING)
+            ->setReceivedAt(new \DateTime('now'))
```

Comment:
I would use `new \DateTimeImmutable()` here (it's recommended by PHP itself (https://www.php.net/manual/en/class.datetime.php) and in general makes handling dates much less error-prone).

Reply:
Updated.